rule/utils.go

```diff
@@ -165,3 +165,9 @@ func checkNumberOfArguments(expected int, args lint.Arguments, ruleName string)
 		panic(fmt.Sprintf("not enough arguments for %s rule, expected %d, got %d. Please check the rule's documentation", ruleName, expected, len(args)))
 	}
 }
+
+var directiveCommentRE = regexp.MustCompile("//(line |extern |export |[a-z0-9]+:[a-z0-9])") // see https://go-review.googlesource.com/c/website/+/442516/1..2/_content/doc/comment.md#494
```

Comment:
I wonder if it needs to be anchored (`^`); I did a quick try using the comment below, and it didn't flag that as invalid;

```go
//this is a regular command that's incorrectly formatted //nolint:foobar // because one two three
```


(disclaimer: I only used the function directly, not in context of the whole code, so perhaps there's already some other code handling this)

Reply:
I think you are right, that's why I raised the point

https://github.com/mgechev/revive/pull/986#issuecomment-2104473372

A comment like this may cause issues or being wrongly reported

    // commenting a nolint //nolint:whatever // whatever is a false positive here

Reply:
It was fixed with #988

Reply:
Thanks!